static/third/sorttable/sorttable.js

```diff
@@ -169,13 +169,13 @@ sorttable = {
     for (var i=0; i<table.tBodies[0].rows.length; i++) {
       text = sorttable.getInnerText(table.tBodies[0].rows[i].cells[column]);
       if (text != '') {
-        if (text.match(/^-?[�$�]?[\d,.]+%?$/)) {
+        if (text.match(/^-?[�$�]?[\d,.]+%?$/)) {
```

Comment:
This looks like an encoding error on save, rather than a change that ought to be made